ERP_INES/Data/Modules/Finance/Repositories/Concrete/PsqlTransactionRepository.cs

```diff
@@ -40,6 +35,34 @@ public async Task<List<Transaction>> GetTransactionsAsync(string? currency)
                 .Where(x => x.PaymentMethod.ISOCurrencySymbol == currency);
         }
 
+        if (!string.IsNullOrWhiteSpace(sort))
+        {
+            switch (sort.ToLower())
+            {
+                case "name":
```

Comment:
#would: for those cases, the logic are been repeated over and over again, sou would be good to make the asc value as default and pass just if I need in desc sorting